README.md

```diff
@@ -33,9 +33,9 @@ showDescription: true
 After you've built your project, `Cockpit.java` file will get generated for you. You can use it like any other code in your project.
 
 <p float="left">
-    <img src="https://github.com/Polidea/Cockpit/blob/master/images/cockpit-full.png" width="216" height="384" />
-    <img src="https://github.com/Polidea/Cockpit/blob/master/images/cockpit-half.png" width="216" height="384" />
-    <img src="https://github.com/Polidea/Cockpit/blob/master/images/cockpit-list.png" width="216" height="384" />
+    <img src="https://github.com/Polidea/Cockpit/blob/readme_update2/images/cockpit-full.png" width="216" height="384" />
+    <img src="https://github.com/Polidea/Cockpit/blob/readme_update2/images/cockpit-half.png" width="216" height="384" />
```

Comment:
will this image survive when branch will be deleted?

Reply:
No, the links need to be updated, but I wanted them to display correctly on the branch.